.travis.yml

```diff
@@ -1,18 +1,20 @@
 language: cpp
+os: osx
+osx_image:
+  - xcode10.1
+  - xcode11.2
+env:
+  global:
+  - MAKEFLAGS="-j2"
+  - Qt5Core_DIR=/usr/local/opt/qt/lib/cmake/Qt5Core
```

Comment:
never needed this, why would i need it now?

Reply:
Because I see in logs that poppler can't find a Qt

Reply:
So poppler-qt isn't built without this.

Reply:
My assumption is that building poppler-qt is in scope of CI of KDE dev :)

Reply:
this is not really "KDE dev" more "poppler dev", but sure, builing most of the code is wanted.